go/consensus/tendermint/apps/scheduler/query.go

```diff
@@ -58,8 +57,8 @@ func (sq *schedulerQuerier) Validators(ctx context.Context) ([]*scheduler.Valida
 	// Since we use flat voting power for now, doing it this way saves
```

Comment:
This comment should be changed/removed as it is wrong now that the state stores a map of public keys to voting power.

Reply:
ok removing